### PR TITLE
Login/auth v3

### PR DESCRIPTION
**Pull request by All-Schuck-Up**
Updated login/ auth feature
  --Using mongoose functionality to link collections

Added onto symptom entry feature
  --Successfully able to post symptom entries to patient collection on submit of form.

### CODE REVIEW

**Comment by asiagromasia** [comment]
Submit button doesn't work hence symptom entry doesn't add to db. No entries in db. I really like your input design with data to be chosen and symptoms to be placed as yes or not.

File: models/Patient.js [hunk 1]
```diff
@@ -1,26 +1,73 @@
...
+      },
+      comment: {
+        type: String,
+      },
+      doctorNote: {
+        type: String,
+      },
+      immediateAttention: {
+        type: Boolean,
+      },
```
Comment (OyunominJuno) [inline models/Patient.js:68]: We did not agree on having the same schema for patients' information and patient entries. Doesn't it change many routes and what we have done already? Also isn't this Yergu's job make it work?

**Comment by All-Schuck-Up** [comment]
We need to discuss state management in the zoom call as the state is being passed around a bunch and there is no need to have so many collection in the database as patient entries are part of the patient profile/record.

**Comment by yergu** [comment]
When I run it on from my Machin I am getting this error "./src/components/patient/Patient.SyptomEntry.js
Module not found: Can't resolve 'semantic-ui-react' in 'C:\workspace\patient-Tracking-Local\AD-410-F\client\src\components\patient'"

**Pull request closed without merging**